dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -176,4 +176,7 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
 stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. When this property is set to true, the application stops starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true. 
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
\ No newline at end of file
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
+
+addStrictTransportSecurityHeader.name=Add the HTTP Strict Transport Security header for HTTPS responses
+addStrictTransportSecurityHeader.desc=Enables the HTTP Strict Transport Security (HSTS) header for HTTPS responses and allows a value for that header to be set. For example: "max-age=31536000;includeSubDomains". HSTS can also be configured in the web.xml via the "com.ibm.ws.webcontainer.ADD_STS_HEADER_WEBAPP" context-param.
```

Comment:
Enables the HTTP Strict Transport Security (HSTS) header for HTTPS responses and allows a value to be set for that header, for example: "max-age=31536000;includeSubDomains". HSTS can also be configured in the web.xml file by setting the "com.ibm.ws.webcontainer.ADD_STS_HEADER_WEBAPP" context parameter.